src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.hpp

```diff
@@ -225,7 +225,11 @@ class CompositionalMultiphaseBase : public FlowSolverBase
     static constexpr char const * elemDofFieldString() { return "compositionalVariables"; }
 
     // inputs
-    static constexpr char const * temperatureString() { return "temperature"; }
+
+    // TODO: when GEOSX becomes thermal, remove "temperatureField" and only keep "temperature"
+    //       for now, let's avoid another change in the xml file
+    static constexpr char const * temperatureString() { return "temperatureField"; }
+    static constexpr char const * inputTemperatureString() { return "temperature"; }
```

Comment:
You could use the key string "temperature" for both, since they are registered on different objects (one on the solver, the other on the mesh), so there would be no conflict. I'd much rather have the actual field be named "temperature" from the get go :)

Reply:
Done, I removed `temperatureField`